application/modules/quotes/views/partial_itemlist_responsive.php

```diff
@@ -0,0 +1,318 @@
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+
+    <?php foreach ($items as $item) { ?>
+      <div class="form-group details-box item">
+				<div class="row">
+					<input type="hidden" name="quote_id" value="<?=$quote_id;?>">
+					<input type="hidden" name="item_id" value="<?=$item->item_id;?>">
+					<input type="hidden" name="item_product_id" value="<?=$item->item_product_id; ?>">
+					<div class="col-xs-12 col-sm-7 col-md-6 col-lg-5">
+						<div class="row">
+							<div class="col-xs-12 col-sm-1">
+								<button type="button" class="btn btn-link up" title="<?php _trans('move_up'); ?>">
+									<i class="fa fa-chevron-up"></i>
+								</button>
+								<button type="button" class="btn btn-link down" title="<?php _trans('move_down'); ?>">
+									<i class="fa fa-chevron-down"></i>
+								</button>
+								<button type="button" class="btn_delete_item btn btn-link" title="<?php _trans('delete'); ?>" data-item-id="<?=$item->item_id;?>">
+									<i class="fa fa-trash-o text-danger"></i>
+								</button>
+							</div>
+
+							<div class="col-xs-12 col-sm-11">
+								<div class="input-group">
+									<label for="item_name_<?=$item->item_id;?>" class="input-group-addon ig-addon-aligned"><?php _trans('item'); ?></label>
+									<input type="text" name="item_name" id="item_name_<?=$item->item_id;?>" class="input-sm form-control" value="<?=_htmlsc($item->item_name);?>">
+								</div>
+								<div class="input-group">
+									<label for="item_description_<?=$item->item_id;?>" class="input-group-addon ig-addon-aligned"><?php _trans('description'); ?></label>
+									<textarea name="item_description" id="item_description_<?=$item->item_id;?>" class="input-sm form-control h135rem"><?=htmlsc($item->item_description);?></textarea>
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="col-xs-12 col-sm-5 col-md-6 col-lg-7">
+						<div class="row">
+							<div class="col-xs-12 col-lg-6">
+								<div class="input-group">
+									<label for="item_quantity_<?=$item->item_id;?>" class="input-group-addon ig-addon-aligned"><?php _trans('quantity'); ?></label>
+									<input type="text" name="item_quantity" id="item_quantity_<?=$item->item_id;?>" class="input-sm form-control" value="<?=format_amount($item->item_quantity);?>" >
+								</div>
+								<div class="input-group">
+										<label for="item_product_unit_id_<?=$item->item_id;?>" class="input-group-addon ig-addon-aligned"><?php _trans('product_unit'); ?></label>
+										<select name="item_product_unit_id" id="item_product_unit_id_<?=$item->item_id;?>" class="form-control input-sm">
+												<option value="0"><?php _trans('none'); ?></option>
+												<?php foreach ($units as $unit) { ?>
+														<option value="<?=$unit->unit_id;?>"
+																<?php check_select($item->item_product_unit_id, $unit->unit_id); ?>>
+																<?php echo htmlsc($unit->unit_name) . "/" . htmlsc($unit->unit_name_plrl); ?>

```

Comment:
Inconsistent use of `<?=` and `<?php echo`.
Nothing troubling but maybe changeable for better style of the code.

Reply:
Just mentioned here once, happens more often

Reply:
changed consistently to long format.